src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -429,21 +435,24 @@ public IVerbHandler<? extends Request> verbHandler()
             Throwable cause = Throwables.getRootCause(e);
             if (cause instanceof Timeout)
             {
+                TxnId txnId = ((Timeout) cause).txnId();
```

Comment:
this was a bug fixed in CASSANDRA-19833 and was easier to keep when I spun off the debug logic (as its one of the other changes to this class)...  I could split this out if desired